AISKULight/Tests/Unit/src/AISKULightSize.Tests.ts

```diff
@@ -2,8 +2,8 @@ import { AITestClass, Assert } from "@microsoft/ai-test-framework";
 import * as pako from "pako";
 
 export class AISKULightSizeCheck extends AITestClass {
-    private readonly MAX_RAW_SIZE = 81;
-    private readonly MAX_BUNDLE_SIZE = 81;
+    private readonly MAX_RAW_SIZE = 82;
```

Comment:
Is this change still needed, looking back the previous test pass seem to pass without this??
After you refactored the code to make it smaller.

If so then it's fine.

Reply:
thanks! It was 81.05 in the cloud test. And after refactoring, it goes down.

Reply:
Turns out that locally it does not exceed 81, but on the cloud it does.